tests/test_dagbuilder.py

```diff
@@ -1158,3 +1160,282 @@ def test_multiple_dependencies(self):
         assert task_names.index("task1") < task_names.index("task3")
         assert task_names.index("task2") < task_names.index("task4")
         assert task_names.index("task3") < task_names.index("task4")
+
+    @pytest.mark.skipif(INSTALLED_AIRFLOW_VERSION.major < 3, reason="Require at least Airflow 3.0.0")
+    def test_asset_schedule(self):
```

Comment:
@pankajastro would it be worth to break the tests into smaller ones, so it is easier to list all possible failures and/or isolate errors?

Reply:
I have tried to split this in commit: 16bb041a112e2fdaf5bb589f2063ecc808c8e534 PTAL

Reply:
Pushed a fix in commit: df35bfae1a9e86ae9ac22cf78807e471c702ef18